lpa-submissions-e2e-tests/cypress/integration/supplementaryDocumentList.feature

```diff
@@ -6,7 +6,6 @@ Feature: As a LPA Planning Officer,
     Given an appeal has been created
```

Comment:
Can the title of this file, and related files, be renames to reflect the proper name of 'uploaded-documents'?

Reply:
updated as per review